api/app/signals/apps/signals/managers.py

```diff
@@ -603,3 +615,24 @@ def update_type(self, data, signal):
                                                                   type=signal_type, prev_type=previous_type))
 
         return signal_type
+
+    def _update_directive_departments_no_transaction(self, data, signal):
```

Comment:
"_update_directive_departments" -> "_update_directing_departments" to match rest of this PR.

Reply:
Forgot to change this to match the rest of the feature, oops 😂